airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/PostgresSource.java

```diff
@@ -56,7 +56,8 @@ public static Source sshWrappedSource() {
   }
 
   PostgresSource() {
-    super(DRIVER_CLASS, new PostgresJdbcStreamingQueryConfiguration(), new PostgresSourceOperations());
+    super(DRIVER_CLASS, new PostgresJdbcStreamingQueryConfiguration(),
+        new PostgresSourceOperations());
```

Comment:
Not sure if these format changes are expected. Better run `./gradle format` before merging.

Reply:
Updated, thanks!
But I always run the "format" command before pushing my changes.